### PR TITLE
[docs] clarify what <path> is

### PR DESCRIPTION
**Pull request by alexfauquette**
From docs feedback

>Path of what? Were the component is used? Root of the project?

sent from https://mui.com/x/migration/migration-pickers-lab/ (from section [2. Run the code mod](https://mui.com/x/migration/migration-pickers-lab/#2-run-the-code-mod))

### CODE REVIEW

File: docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Described below are the steps needed to migrate from v5 to v6.
 
 The `preset-safe` codemod will automatically adjust the bulk of your code to account for breaking changes in v6. You can run `v6.0.0/data-grid/preset-safe` targeting only Data Grid or `v6.0.0/preset-safe` to target Date and Time pickers as well.
 
+You can either run it on specific file, folder, or your entire codebase when choosing the `<path>` argument.
```
Comment (LukasTy) [inline docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md:28]:
```suggestion
You can either run it on a specific file, folder, or your entire codebase when choosing the `<path>` argument.
```

File: docs/data/migration/migration-pickers-lab/migration-pickers-lab.md [hunk 2]
```diff
@@ -58,7 +58,9 @@ Learn more on [the Licensing page](/x/introduction/licensing/#license-key-instal
 
 ### 2. Run the code mod
 
-We have prepared a codemod to help you migrate your codebase
+We have prepared a codemod to help you migrate your codebase.
+
+You can either run it on specific file, folder, or your entire codebase when choosing the `<path>` argument.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-lab/migration-pickers-lab.md:63]:
```suggestion
You can either run it on a specific file, folder, or your entire codebase when choosing the `<path>` argument.
```

**Review by LukasTy — COMMENTED** [review]
Shouldn't we also make the same changes on the https://deploy-preview-8764--material-ui-x.netlify.app/x/migration/migration-pickers-v5/#run-codemods page? 🤔

**Comment by alexfauquette** [comment]
> Shouldn't we also make the same changes on the [deploy-preview-8764--material-ui-x.netlify.app/x/migration/migration-pickers-v5/#run-codemods](https://deploy-preview-8764--material-ui-x.netlify.app/x/migration/migration-pickers-v5/#run-codemods) page? thinking

I'm wondering if adding this extra info could be confusing with the following instruction:

> It should be only applied once per folder.

**Comment by LukasTy** [comment]
> I'm wondering if adding this extra info could be confusing with the following instruction:
> 
> > It should be only applied once per folder.

IMHO, this particular section should/cold be removed as it is somewhat pointless and was already removed from the data grid documentation. 🤔 
Technically, running the codemod multiple times shouldn't break anything (make any further changes)...

**Review by joserodolfofreitas — APPROVED** [review]

**Pull request merged**